New package: ElectronWraper v1.0.0

JuliaRegistrator opened the pull request:
- Registering package: ElectronWraper
- Repository: https://github.com/anke1460/ElectronWraper
- Created by: @anke1460
- Version: v1.0.0
- Commit: d25464b04b567c1a2e309b552379e163e4b9de35
- Git reference: main

goerz commented:
https://github.com/JuliaRegistries/General/pull/103254#issuecomment-2008622906:

>> Also: how does this package relate to https://github.com/davidanthoff/Electron.jl?
>
> yes, it fork Electron.jl, and I update to electron latest version

In most circumstances, forks of existing packages are not suitable for registration. You should try to contribute your changes to `Electron.jl`

JuliaHubRegistrator committed:
New package: ElectronWraper v1.0.0

UUID: 389417e1-e013-410a-9897-1150a2d1e8b9
Repo: https://github.com/anke1460/ElectronWraper.git
Tree: 0da17e609bb0d56ce8390b438dcc769d0a318f11

Registrator tree SHA: 7242ef64be5953fac9bf8b3efa3a7c4d2d44ca09

cormullion commented:
The spelling mistake in the name is unfortunate.

goerz commented:
Yes, that would have to be fixed.

But a fork of an existing package should not be registered anyway. Although I’m not sure this really is a fork.

@davidanthoff, can you have a look at this?

PallHaraldsson commented:
It seems to be a fork (and though should be objected to), I at least see "FORK Electron.jl and update to electron V29.1.4"

@anke1460 if this is just a trivial update then help get it into current Electron.jl. For just you, and even others you could use the package as unregistered, and just rename it back to Electron. I don't find it better to add Wrapper (without the type) or Fork ending. In extreme cases a package needs to be forked, and we should likely have a policy about registering such packages, and discussion about best names to use then. For open source it needs to be a "nuclear option" (seemingly no needed here, see e.g. XEmacs fork case and SXEmacs fork of fork), but not all will agree, so it NEEDS not be accented for registration.
